### PR TITLE
Fit-it-day placeholder update for Deploy to Production (Van)

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Fit-it-day placeholder ($) update for Deploy to Production
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-729831](https://oktainc.atlassian.net/browse/OKTA-729831)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/java/deployment.md](https://github.com/okta/okta-developer-docs/blob/8f03d6eb7bca45042e101332da200d6d062430e9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjava%2Fdeployment.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/50f403a1-7478-48a1-b0be-dbc928738ae1) | 83  | 100  | 54  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a7466282-2c1d-43de-9b50-1b17cc3e8dde?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/java/deployment.md](https://github.com/okta/okta-developer-docs/blob/8e7dc707025fb87e891017a59f29d3786af9f4f9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjava%2Fdeployment.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/04118286-54d9-49b4-bb94-220e88f2982f) | 83  | 100  | 54  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md](https://github.com/okta/okta-developer-docs/blob/8e7dc707025fb87e891017a59f29d3786af9f4f9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-bulk%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c54c9c03-832a-44d7-82b2-8cdd26ee9f28) | 100  | 90  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-password-hooks/main/index.md](https://github.com/okta/okta-developer-docs/blob/8e7dc707025fb87e891017a59f29d3786af9f4f9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-password-hooks%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/87efb64e-b5ec-490e-b0cf-8fe0fe7fafb3) | 90  | 64  | 75  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.5a0b7c57-f794-406f-a2e4-f4ebfa36dbfe?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/java/deployment.md](https://github.com/okta/okta-developer-docs/blob/707a72da507f8f60da5773d32efa4cf127878e15/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjava%2Fdeployment.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b3502d66-1253-4ee3-86f5-c7ddabacb87f) | 100  | 82  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md](https://github.com/okta/okta-developer-docs/blob/707a72da507f8f60da5773d32efa4cf127878e15/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-bulk%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d1da8df4-4cd3-4c90-aacf-b030c66bdcde) | 100  | 90  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-password-hooks/main/index.md](https://github.com/okta/okta-developer-docs/blob/707a72da507f8f60da5773d32efa4cf127878e15/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-password-hooks%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/24ec7245-216d-4eb8-bfc4-de6f6b98b999) | 90  | 64  | 75  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.29115d7c-60a7-45d6-ad8b-c59f83e2a08a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/java/deployment.md](https://github.com/okta/okta-developer-docs/blob/fd05c6d842bbc016b8a7a6ba3f8814d5b7d089a6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjava%2Fdeployment.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/890ceca1-a2c6-4b7c-83df-1c9f40943cdf) | 100  | 82  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md](https://github.com/okta/okta-developer-docs/blob/fd05c6d842bbc016b8a7a6ba3f8814d5b7d089a6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjavascript%2Fdeployment.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7bdc6e90-d0e5-4e4e-b50a-818a24c62284) | 100  | 55  | 65  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/reactnative/deployment.md](https://github.com/okta/okta-developer-docs/blob/fd05c6d842bbc016b8a7a6ba3f8814d5b7d089a6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Freactnative%2Fdeployment.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0ddf9aa1-3ec1-47a5-9e8b-a114ffb12d65) | 15  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md](https://github.com/okta/okta-developer-docs/blob/fd05c6d842bbc016b8a7a6ba3f8814d5b7d089a6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-bulk%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2b24b357-2f4a-4cee-9e1e-5f5b8eda59d4) | 100  | 90  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-password-hooks/main/index.md](https://github.com/okta/okta-developer-docs/blob/fd05c6d842bbc016b8a7a6ba3f8814d5b7d089a6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-password-hooks%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e8fe9d5b-9fa7-4338-b744-d5a916408b1d) | 90  | 64  | 75  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.fd798389-1432-4455-874a-fe79e4f29954?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## Deploying a JavaScript app
+## Deploy a JavaScript app
 
-JavaScript applications, when built for production, are just a bunch of static files. You can deploy them to any web server, configure that web server to be SPA-aware, and you should be good to go!
+JavaScript apps built for production are a collection of static files. You can deploy them to any web server, configure that web server to be SPA-aware, and you should be good to go.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md:3]:
```suggestion
JavaScript apps built for production are a collection of static files. You can deploy them to any web server, and then configure that web server to be SPA-aware.
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md:3]: not sure if something like "and then you're ready to use them" is a reasonable replacement for "should be good to go" but if not I think it's fine to just cut. (not sure how well it translates)

File: packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/reactnative/deployment.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-## Deploying a React Native app
+## Deploy a React Native app
 
-Mobile applications, when built for production, are usually a binary file that you upload to an app store.
+Mobile apps built for production are usually a binary file that you upload to an app store.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/reactnative/deployment.md:3]:
```suggestion
A mobile app built for production is usually a binary file that you upload to an app store.
```

**Review by grahamsmith-okta — CHANGES_REQUESTED** [review]
couple small comments/suggestions

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/java/deployment.md](https://github.com/okta/okta-developer-docs/blob/7c6e0f88bbb8b1ef536aedde3e6707e88dcb0c0e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjava%2Fdeployment.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0425df2f-ed4f-4864-a015-2626682aca0b) | 100  | 82  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md](https://github.com/okta/okta-developer-docs/blob/7c6e0f88bbb8b1ef536aedde3e6707e88dcb0c0e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjavascript%2Fdeployment.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f1ee1351-c652-4a7b-9f0b-2c31c41b4d26) | 100  | 55  | 65  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/reactnative/deployment.md](https://github.com/okta/okta-developer-docs/blob/7c6e0f88bbb8b1ef536aedde3e6707e88dcb0c0e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Freactnative%2Fdeployment.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94d62601-10d0-4015-956c-a20190b8d049) | 15  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md](https://github.com/okta/okta-developer-docs/blob/7c6e0f88bbb8b1ef536aedde3e6707e88dcb0c0e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-bulk%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ae848352-3b6d-4284-bcc9-729a54ba33bd) | 100  | 90  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-password-hooks/main/index.md](https://github.com/okta/okta-developer-docs/blob/7c6e0f88bbb8b1ef536aedde3e6707e88dcb0c0e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-password-hooks%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/26b1d157-ac65-4558-af34-49ea56861a82) | 90  | 64  | 75  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.97847f37-fd6b-42e1-8d53-f68d8bfa0587?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/java/deployment.md](https://github.com/okta/okta-developer-docs/blob/7ba80a0d90ac72e1fd6e489bc586a8443285f428/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjava%2Fdeployment.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1ddc5d52-80a8-4436-93b2-b0266cafc539) | 100  | 82  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md](https://github.com/okta/okta-developer-docs/blob/7ba80a0d90ac72e1fd6e489bc586a8443285f428/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Fjavascript%2Fdeployment.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6dd5013d-253a-4cc9-9096-2514ae6d55d8) | 100  | 55  | 64  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/reactnative/deployment.md](https://github.com/okta/okta-developer-docs/blob/7ba80a0d90ac72e1fd6e489bc586a8443285f428/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdeploy-your-app%2Fmain%2Freactnative%2Fdeployment.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6bed8496-17e8-4ea9-ae2e-19f1720a2f03) | 15  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md](https://github.com/okta/okta-developer-docs/blob/7ba80a0d90ac72e1fd6e489bc586a8443285f428/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-bulk%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e939ce4f-d3b6-4e12-b2d0-248a12957b22) | 100  | 90  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/migrate-to-okta-password-hooks/main/index.md](https://github.com/okta/okta-developer-docs/blob/7ba80a0d90ac72e1fd6e489bc586a8443285f428/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmigrate-to-okta-password-hooks%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2f6323ef-e824-4ce5-a612-f4a6d6b2eec4) | 90  | 64  | 75  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f23d127b-d2e7-4c8f-b314-232bcbd0b42b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**